src/components/ListItem.jsx

```diff
@@ -2,9 +2,28 @@ import { useState } from 'react';
 import './ListItem.css';
 import { updateItem, deleteItem } from '../api';
 import { calculateDateNextPurchased, ONE_DAY_IN_MILLISECONDS } from '../utils';
+import {
+	Restore as OverdueIcon,
+	RestartAlt as SoonIcon,
+	RadioButtonUnchecked as KindOfSoonIcon,
+	RemoveCircle as NotSoonIcon,
+	RadioButtonChecked as InactiveIcon,
+} from '@mui/icons-material';
 
 const currentDate = new Date();
 
+const urgencyStatusIcons = {
+	overdue: OverdueIcon,
+	soon: SoonIcon,
+	kindOfSoon: KindOfSoonIcon,
+	notSoon: NotSoonIcon,
+	inactive: InactiveIcon,
+};
+
+const urgencyStatusWidth = {
+	fontSize: '1.5em',
+};
```

Comment:
This constant name suggests it controls the width, but it actually sets the fontSize?

Reply:
very fair, I will fix this!